### PR TITLE
Merge with changes to Help

### PR DESCRIPTION
**Pull request by MrDraker**
help.hbs:
  added a partway done accordion and some questions to give an idea of what the end result would look like

globals.json
  added a faq dictionary that works with help.hbs

### CODE REVIEW

**Comment by MrDraker** [comment]
Thinks everything is okay now. I edited globals.json so they could merge but there is stuff in /templates that I didn't touch that is saying there is a conflict.

File: data/globals.json [hunk 1]
```diff
@@ -74,7 +74,22 @@
           "title": "FAQ",
           "path": "faq.html"
         }
-      }
+      },
+      "questions": [
```
Comment (hwkr) [inline data/globals.json]: Should this not be under FAQ?

File: templates/help/faq.hbs [hunk 2]
```diff
@@ -8,6 +8,30 @@
...
+                <div id="collapseOne" class="panel-collapse collapse in" role="tabpanel" aria-labelledby={{@index}}: {{this}}>
+                    <div class="panel-body">
+                        {{answer}}
+                    </div>
+                </div>
+            </div>
+        {{/each}}
+    </div>
+    {{! Access the page context with `user` (see amy.json)}}
+    <p>{{user.content}}</p>
```
Comment (hwkr) [inline templates/help/faq.hbs]: I don't think we need these user contents

File: templates/help/faq.hbs [hunk 3]
```diff
@@ -8,6 +8,30 @@
     }}
     {{! Access the page context with `page` (see globals.json)}}
     <h2>{{page.title}}</h2>
+    {{!This is for the collapsible lists}}
+    <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+        {{#each sections.help.questions}}
```
Comment (hwkr) [inline templates/help/faq.hbs]: This can refer to the page context

**Comment by hwkr** [comment]
The collapses aren't working for me, I think the ID's might be broken.

File: templates/help/faq.hbs [hunk 4]
```diff
@@ -8,6 +8,30 @@
...
+            <div class="panel panel-default">
+                <div class="panel-heading" role="tab" id={{@index}}: {{this}}>
```
Comment (hwkr) [inline templates/help/faq.hbs:15]: This isn't how you should give an id see http://www.w3schools.com/tags/att_global_id.asp

**Pull request merged**